functions/count_char.py

```diff
@@ -0,0 +1,40 @@
+def count_char(string,char):
+	"""function to count the given character from given string.
+
+    Args:
+       string(str): The first parameter.
+       char(char): The second parameter.
+
+    Returns:
+        -1: The return value, when entered charecter is not present in string.
+        count: Number of count given charater in given string.
+
+    Example:
+    	string='hello selva',char='a'     | return count | 1
+    	string='hello selva',char='x'     | return -1
+    	
+    """	
+	if not string:
```

Comment:
Validation not required, since you have documented that what argument user should pass.